src/main/ontop/1.0/GenericPropertyOntology-1.0.ttl

```diff
@@ -386,7 +386,7 @@ seas:airQuality a owl:ObjectProperty , owl:FunctionalProperty ;
   seas:AirCO2LevelProperty a owl:Class ;
   rdfs:label "Air CO2 level Property"@en ;
   skos:prefLabel "Air Carbon dioxide level Property"@en;
-  rdfs:comment """Air CO2 (Carbon dioxide) level properties is an Air Quality Property which measure the level of CO2 gas in the  the Air."""@en ;
+  rdfs:comment """Air CO2 (Carbon dioxide) level properties is an Air Quality Property which measure the level of CO2 gas in the Air."""@en ;
```

Comment:
change everywhere:

properties -> property
and which measure -> which measures